docs/user/howto/lintchart.md

```diff
@@ -37,7 +37,7 @@ $ hypper lint cmd/hypper/testdata/testcharts/vanilla-helm
 
 Running against a shared-dependencies annotation that is malformed will emit an `[ERROR]`:
 
-```shell
+```terminal
```

Comment:
:arrow_up: stopping comments on this, or I will be annoying :)

Reply:
resolving, let's leave the other conversation.